App/Sagas/StorageSagas.ts

```diff
@@ -39,7 +39,7 @@ export function * refreshLocalImages () {
         const lastRefresh = yield select(StorageSelectors.lastPhotoRefresh)
         // update last time checked to now
         const currentRefresh = (new Date()).getTime()
-        yield call(CameraRoll.requestLocalPhotos, lastRefresh)
+        const results: LocalPhotoResult[] = yield call(CameraRoll.requestLocalPhotos, lastRefresh)
```

Comment:
Cool, `requestLocalPhotos` now works like this. Will keep moving forward with it in the morning.

Reply:
super!

Reply:
"move forward with it" do you intend to do that on this branch, or merge and continue?

Reply:
On this branch